### DISCUSSION TITLE
Cannot find recipe/cachetool.php

### DISCUSSION THREAD

**Discussion by jasperf** [deployphp/deployer#2200] [Help needed]
It seems that `vendor/deployer/deployer/recipe/` does not have cachetool `cachetool.php anymore. Is that correct? It used to be there but on latest `composer update` post `composer remove deployer/recipes` as recipes package is no longer used I now cannot find this recipe anymore. Laravel recipe is still there.

**Comment by antonmedv** [comment]
https://github.com/deployphp/deployer/blob/master/contrib/cachetool.php

**Comment by jasperf** [comment]
Thanks for that. I managed to revert to older version for now. And when I use the new one I will point to this new location.

**Comment by Tchiller** [comment]
How can I inclue that? I did a `require 'contrib/cachetool.php';` with version 6.8.0 and get 
```
Warning: require(contrib/cachetool.php): failed to open stream: No such file or directory ...
Fatal error: require(): Failed opening required 'contrib/cachetool.php' ...
```

**Reply by antonmedv** [reply]
The `contrib` available only with v7 which should be released soon. Now you can use v7-beta.

**Reply by Tchiller** [reply]
Thank you, @antonmedv I will try that now.
